pkg/cqrs/base_cqrs/cqrs.go

```diff
@@ -80,6 +82,207 @@ func (w wrapper) dialect() string {
 	return "sqlite3"
 }
 
+func (w wrapper) GetSpansByRunID(ctx context.Context, runID ulid.ULID) (*cqrs.OtelSpan, error) {
```

Comment:
This is the roll-up performed when fetching from the store. It'll be implemeneted again for Cloud.